io.missioncenter.MissionCenter.json

```diff
@@ -0,0 +1,232 @@
+{
+  "app-id": "io.missioncenter.MissionCenter",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "44",
+  "sdk": "org.gnome.Sdk",
+  "command": "missioncenter",
+  "build-options": {
+    "no-debuginfo": true
+  },
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--device=all",
+    "--socket=wayland",
+    "--talk-name=org.freedesktop.Flatpak",
+    "--talk-name=org.gnome.Settings",
+    "--system-talk-name=org.freedesktop.systemd1",
+    "--system-talk-name=org.freedesktop.NetworkManager",
+    "--filesystem=xdg-data/flatpak/exports/share:ro",
+    "--filesystem=/var/lib/flatpak/exports/share:ro",
+    "--filesystem=xdg-data/flatpak/app:ro",
+    "--filesystem=/var/lib/flatpak/app:ro",
+    "--env=XDG_DATA_DIRS=/app/share:/usr/share:/usr/share/runtime/share:/run/host/user-share:/run/host/share:/var/lib/flatpak/exports/share:~/.local/share/flatpak/exports/share"
+  ],
+  "cleanup": [
+    "/sbin",
+    "/etc",
+    "/sbin",
+    "/include",
+    "/lib/pkgconfig",
+    "/lib/python*",
+    "/man",
+    "/share/doc",
+    "/share/gtk-doc",
+    "/share/man",
+    "/share/pkgconfig",
+    "*.la",
+    "*.a"
+  ],
+  "modules": [
+    {
+      "name": "rust-toolchain-x86_64",
```

Comment:
FWIW, there's also an extension with Rust available.

Reply:
I need to use a version of Rust that has both x86_64-unknown-linux-gnu and x86_64-unknown-linux-musl and the two are exact same version (1.70.0 for example); the default one provided by the sdk extension does not have musl
I need a version with musl to build a static binary that is executed outside the sandbox. This binary gathers data from /proc.